### DISCUSSION TITLE
How to deploy in Android

### DISCUSSION THREAD

**Discussion by ybr990629** [eunomia-bpf/eunomia-bpf#309] [General]
我的Android设备是ARM64的，根据文档并不能够在我的设备里部署 下载后会报错：
bash: ./ecli: cannot execute binary file: Exec format error

**Comment by Officeyutong** [comment]
Could you please provide `uname -a` output?

**Reply by ybr990629** [reply]
Linux localhost 5.15.94-android13-xxxxx-aarch64 GNU/Linux

**Reply by Officeyutong** [reply]
Did you downloaded the aarch64 ecli? According to the filename, you seem to have downloaded the x86 one

**Reply by Officeyutong** [reply]
aarch64 ones could be found at https://github.com/eunomia-bpf/eunomia-bpf/releases/download/v1.0.11/ecli-aarch64, they have filename `ecli-aarch64`

**Reply by JiaHuann** [reply]
BTW，you need to use ecc-arch to build package.json ,otherwise ecli may not run as well. And we are trying to release a ecc-arch with full-env packed.

**Reply by ybr990629** [reply]
Oh! It's my fault, I’m not found releases address :)
Thanks
